addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/utils/HttpUtils.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.squeezebox.internal.utils;
+
+import java.net.URL;
+import java.net.URLConnection;
+
+import org.apache.commons.httpclient.HttpClient;
+import org.apache.commons.httpclient.HttpStatus;
+import org.apache.commons.httpclient.methods.PostMethod;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+/**
+ * Collection of methods to help retrieve HTTP data from a SqueezeServer
+ * 
+ * @author Dan Cunningham
+ *
+ */
+public class HttpUtils {
+	private static Logger logger = LoggerFactory.getLogger(HttpUtils.class);
+	private static int TIMEOUT = 5000;
+	/**
+	 * JSON request to get the CLI port from a Squeeze Server
+	 */
+	private static final String JSON_REQ = "{\"params\": [\"\", [\"pref\" ,\"plugin.cli:cliport\",\"?\"]], \"id\": 1, \"method\": \"slim.request\"}";
+
+	/**
+	 * Simple logic to perform a post request
+	 * 
+	 * @param url
+	 * @param timeout
+	 * @return
+	 */
+	public static String post(String url, String postData) throws Exception {
+		HttpClient client = new HttpClient();
+		PostMethod method = new PostMethod(url);
+		method.setRequestBody(postData);
+		method.getParams().setSoTimeout(TIMEOUT);
+		try {
+			int statusCode = client.executeMethod(method);
+			if (statusCode != HttpStatus.SC_OK) {
+				logger.error("Method failed: {}", method.getStatusLine());
```

Comment:
throwing the exception should be enough - there should not be an additional error logging